.github/workflows/build.yml

```diff
@@ -124,7 +132,7 @@ jobs:
             })
 
       - name: Checkout
-        if: ${{ steps.check_ci_changes.outputs.run-tests != 'true' }}
+        if: ${{ steps.check_ci_changes.outputs.run-tests != 'true' && steps.check_ci_changes.outputs.do-not-build != 'true' && steps.check_ci_changes.outputs.do-not-build != 'true' }}
```

Comment:
You are missing that sometimes i get over zealous with cut and paste :-)

Also I did think about combining these into one after they are used to output correct messages, but then worried it might actually complicate things more.  Let me know if you would prefer that - would at least save these long lines.

Reply:
Combined into one now:
steps.check_build_required.outputs.run-build == 'true'